_commands/toys/t.md

```diff
@@ -0,0 +1,219 @@
+
+### Using t
+
+#### Help command
+
+Information on t is available through its help command.
+
+~~~ bash
+$ t help
+Commands:
+  t accounts                          # List accounts
+  t authorize                         # Allows an application to request user authorization
+  t block USER [USER...]              # Block users.
+  t delete SUBCOMMAND ...ARGS         # Delete Tweets, Direct Messages, etc.
+  t direct_messages                   # Returns the 20 most recent Direct 
+...
+
+Options:
+  -C, [--color=COLOR]   # Control how color is used in output
+                        # Default: auto
+                        # Possible values: icon, auto, never
+  -P, [--profile=FILE]  # Path to RC file
+                        # Default: /home/alexander/.trc
+~~~
+
+You can also get help for any specific t command.
+
+~~~ bash
+$ t help stream
+Commands:
+  t stream all                          # Stream a random sample of all Tweets (Control-C to stop)
+  t stream help [COMMAND]               # Describe subcommands or one specific subcommand
+  t stream list [USER/]LIST             # Stream a timeline for members of the specified list (Control-C...
+...
+~~~
+
+
+#### Posting tweets and DMs
+Post a tweet from the terminal.
+
+~~~ bash
+$ t update "Tweet tweet"
+~~~
+
+Tweet at a user.
+
+~~~ bash
+$ t update "@chezbetty When will Soylent be back in stock? Bundaberg?" 
+~~~
+
+Send an ROT13 encoded direct message.
+Note: stringing commands together makes t *really powerful*. (And, in this case, *really confusing* to T Swizzle.)
+
+~~~ bash
+$ alias rot13="tr '[A-Za-z]' '[N-ZA-Mn-za-m]'"
+$ echo "You belong with me" | rot13 | xargs -0 t dm @SwitfOnSecurity
+~~~
+
+
+
+#### See what's happening in your world
+Print the 20 most recent tweets on your timeline.
+
+~~~ bash
+$ t timeline
+~~~
+
+Variation: Print a variable number of tweets from your timeline.
+
+~~~ bash
+$ t timeline -n 5
+~~~
+
+Stream your timeline in realtime. Type ^C to exit.
+
+~~~ bash 
+$ t stream timeline
+~~~
+
+Variation: Default streaming looks pretty but isn't very illuminating. Streaming in long mode shows much more info. Let's order it by the newest user to 
+
+~~~ bash
+$ t stream timeline -l
+~~~
+
+See what's trending in the Ann Arbor area. (<a href="http://woeid.rosselliot.co.nz/lookup/ann%20arbor">Uses the Ann Arbor WOEID.</a>)
+
+~~~ bash
+$ t trends 2354842
+~~~
+
+#### Random Power
+
+Favorite the most recent tweet by What-If Numbers.
+
+~~~ bash
+$ t timeline @whatifnumbers -n 1 -l | awk '{print $1}' | xargs t favorite
+~~~
+
+Block a user.
+
+~~~ bash
+$ t block @realDonaldTrump
+~~~
+
+Note: The above command does not prevent the user from running for president. Should such an election occur, report them to Twitter for abuse of power. 
```

Comment:
That's totally fair. 

I picked the most benign handles I could think of to replace them with. I considered removing those two examples altogether, but decided against it because they're important command actions to cover. I can remove them if it's still an issue.